.eslintrc

```diff
@@ -0,0 +1,127 @@
+env:
```

Comment:
How difficult is it to pack these rules as an `asyncapi` eslint plugin? I think it would be cool to have one because it's honestly getting painful 😅

Reply:
It shouldn't be hard, just create an npm package, but first we need to figure out what rules we should have in there. I will create issue (or maybe discussion) for that in the community repo.

Reply:
👍